fourth-wall.css

```diff
@@ -81,6 +81,9 @@ body {
 #pulls li.thumbsup {
     background: #5c9317;
 }
+#pulls li.changes-requested {
+    background: #BF1E2D !important;
```

Comment:
Yeah, in the query string, documented here: https://github.com/alphagov/fourth-wall/tree/d6942511a5c37c94a4466967133139ef686a6618#how-to-use